filetests/isa/x86/optimized-zero-constants-32bit.clif

```diff
@@ -16,3 +16,18 @@ ebb0:
   return v1
 }
 
+function %zero_dword() -> i32 fast {
+ebb0:
+  ; asm: xor %eax, %eax
+  [-,%rax]     v0 = iconst.i32 0       ; bin: 31 c0
+  ; asm: xor %edi, %edi
+  [-,%rdi]     v1 = iconst.i32 0       ; bin: 31 ff
+  return v0
+}
+
+function %zero_byte() -> b1 fast {
+ebb0:
+  ; asm: xor %eax, %eax
+  [-,%rax]     v1 = bconst.b1 false       ; bin: 30 c0
```

Comment:
if the `b1` false recipe is something to keep, I'll copy this into the 64bit tests as well. If not, I'll delete it